app/templates/components/nav-bar.hbs

```diff
@@ -54,12 +54,12 @@
   </div>
 </div>
 <div class="mobile only row">
-  <div class="ui navbar menu">
+  <div class="ui navbar {{if device.isMobile 'tiny'}} menu">
     <a class="menu item open sidebar">
-      <i class="large content icon"></i>
+      <i class="small content icon"></i>
```

Comment:
Keep it just `content icon` don't make it small, it gets too small and I tested the navbar doesn't overflow without the small class.